### DISCUSSION TITLE
Clonacìon de pagina

### DISCUSSION THREAD

**Discussion by jmlimas** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#334] [General]
https://dribbble.com/shots/16001939-Vaccination-Vaccine-landing-page-website

esta  pagina  no se  puede  bajar  gratis para ser  modificada  de  donde la  bajo..???

existe  algún repositorio para  poderla  bajar..??

**Comment by martinmurillo2020** [comment]
Es que la idea es que lo clones vos usando tu propia imaginación y conocimientos.

**Comment by JonathanJRodriguez** [comment]
Hola,

Asi como lo dice Martín, la idea es que a partir de la imagen, tú hagas una página de cero que se vea igual.

Saludos!

**Comment by Ingrzs** [comment]
Pues dale inspeccionar pagina y ahi estará el código, como dicen arriba la idea es que tu mismo clones la pagina desde cero porque en la vida laboral te piden trabajos en los que clientes quieren una pagina igual a tal
